test/test-execute/exec-specifier.service

```diff
@@ -0,0 +1,15 @@
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/test %n = exec-specifier.service
+ExecStart=/usr/bin/test %N = exec-specifier
+ExecStart=/usr/bin/test %p = exec-specifier
+ExecStart=/usr/bin/test %P = exec/specifier
+ExecStart=/usr/bin/test %f = /exec/specifier
+ExecStart=/usr/bin/test %t = /run
+ExecStart=/usr/bin/test %S = /var/lib
+ExecStart=/usr/bin/test %C = /var/cache
+ExecStart=/usr/bin/test %L = /var/log
+ExecStart=/usr/bin/test %U = 0
```

Comment:
In the discussion in PR #7062, @evverx encourages to use `/bin/sh -c`. Do you agree with this?

Reply:
not really. there are reasons to enclose things in sh, but it makes things more unreadable (and slower), hence I'd usually avoid it, really, unless there's a good reason to have it.